package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "3.0.4",
+  "version": "4.0.0-alpha.0",
```

Comment:
Alpha-version packages should probably have the following in their package.json:
```
"publishConfig": {
  "tag": "alpha"
}
```
This means they will be published with an `alpha` distribution tag. I.E. `npm publish --tag alpha`.

Reply:
Thanks @simonsinclair. I'll update these now

Reply:
@simonsinclair done.

Reply:
Nice one, Rowland!